.github/workflows/build-docs.yml

```diff
@@ -16,15 +16,15 @@ jobs:
         uses: actions/setup-python@v1
         with:
           python-version: "3.7"
-      - name: Install Flit
-        run: python3.7 -m pip install flit
+      - name: Install Poetry
+        run: python3.7 -m pip install poetry
```

Comment:
I recommend pinning Poetry version here to get truly reproducible builds, and avoid issues if/when lockfile format changes or similar issues.

Reply:
Here I'm not sure, because I followed the same idea that was already in place.